frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/bc.ts

```diff
@@ -0,0 +1,128 @@
+import * as _ from "lodash";
+export const MOCK_SECRET = 'test-secret-for-bs';
+
+//const tier1TableCheckBoxes = cy.get('table[aria-label="Select Backing Store for Tier 1"] input');
+//const tier2TableCheckBoxes = cy.get('table[aria-label="Select Backing Store for Tier 2"] input');
+
+export enum Tier {
+  SPREAD = 'SPREAD',
+  MIRROR = 'MIRROR',
+}
+
+const TierCountMap = Object.freeze({
+  [Tier.SPREAD]: 1,
+  [Tier.MIRROR]: 2,
+});
+
+export const cleanup = () => {
+  cy.exec('oc delete secret test-secret-for-bs -n openshift-storage', { failOnNonZeroExit: false });
+  cy.exec('oc delete backingstore test1 test2 test3 test4 -n openshift-storage', { failOnNonZeroExit: false });
+  cy.exec('oc delete backingstore noobaa-default-backing-store -n openshift-storage', { failOnNonZeroExit: false });
+  cy.exec('oc delete bucketclass test-affwe-bucketclass -n openshift-storage', { failOnNonZeroExit: false });
+}
+
+export const mockDefaultBS = {
+  apiVersion: 'noobaa.io/v1alpha1',
+  kind: 'BackingStore',
+  metadata: {
+    name: 'noobaa-default-backing-store',
+  },
+  spec: {
+    awsS3: {
+      region: 'us-east-1',
+      secret: {
+        name: MOCK_SECRET,
+        namespace: 'openshift-storage',
+      },
+      targetBucket: 'ssd',
+    },
+    ssl: false,
+    type: 'aws-s3',
+  },
+}
+
+export const createDummyBackingStore = (name: string) => {
```

Comment:
Let's create PVC based Backing Stores we could verify that those BackingStores are actually ready. And later on, verify that the BucketClass also reaches the `Ready` state.

Reply:
Sure